Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:18
```

Comment:
Does this Docker image run locally?  Reason I ask is I had to do a `yarn build` and then [copy over specific folders](https://github.com/cds-snc/platform-forms-client/blob/develop/Dockerfile.pr#L45-L46) to get the Form's PR environment work.  But, that could also just be tied to Forms and not needed for all Next.js projects.

Reply:
I did not have any issue running it locally